.circleci/config.yml

```diff
@@ -111,15 +111,15 @@ workflows:
             - build
   deploy:
     jobs:
-      - deploy_win64:
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              only: /[0-9]+(\.[0-9]+)+.*/
-      - deploy_linux:
-          filters:
-            branches:
-              ignore: /.*/
-            tags:
-              only: /[0-9]+(\.[0-9]+)+.*/
+      - deploy_win64
```

Comment:
why comment code that is not being used? just remove it. much cleaner

Reply:
This is actually very temporary and I intend to use these filters moving forward... Just wanted them commented out to kick off a build in circleCI - I can delete them from this branch if its preferable though